packages/gatsby-plugin-offline/src/sw-append.js

```diff
@@ -1,7 +1,3 @@
-/* global importScripts, workbox, idbKeyval */
-
-importScripts(`idb-keyval-iife.min.js`)
```

Comment:
Rather than prepending the whole `importScripts` line, perhaps you could only replace the url, like is being done for `%pathPrefix%` and `%appFile%`?

Reply:
Yeah, that's better. `importScripts` must be in `sw-append.js`, otherwise the build fails.